src/org/jitsi/impl/neomedia/rtp/ResumableStreamRewriter.java

```diff
@@ -79,7 +72,7 @@ public ResumableStreamRewriter()
      * @param timestampDelta The timestamp delta between what's been accepted
      * and what's been received, mod 2^32.
      */
-    public ResumableStreamRewriter(
+    private ResumableStreamRewriter(
```

Comment:
Right, this class is now only used in DiscardTransformEngine to mask gaps explicitly caused by us. I'll remove the obsolete ctors.